lib/web3/property.js

```diff
@@ -76,7 +81,7 @@ Property.prototype.extractCallback = function (args) {
  */
 Property.prototype.attachToObject = function (obj) {
     var proto = {
-        get: this.get.bind(this),
+        get: this.buildGet() 
```

Comment:
it was not moved, cause it always was it has always been it's own function. We just do not do binding here.

Also, I find it more readable (and possibly testable), when it's separated.